R/dummy.R

```diff
@@ -258,85 +258,75 @@ warn_new_levels <- function(dat, lvl, details = NULL) {
 
 #' @export
 bake.step_dummy <- function(object, new_data, ...) {
-  check_new_data(names(object$levels), object, new_data)
-
-  # If no terms were selected
-  if (length(object$levels) == 0) {
```

Comment:
It looks like we didn't re-introduce the early return that was here?

Reply:
Not needed since we are looping over the selected variables https://github.com/tidymodels/recipes/pull/1182/files#diff-db374e7fd2fd247f43cade403bf9b66b0abad4ff7dffc9025b2f06e45d660fd0R270

Reply:
Feels worth not touching the exit handlers unless needed though, no?

Reply:
true, will put it back in